pavelib/assets.py

```diff
@@ -126,7 +178,7 @@ def register(self, observer):
         """
         register files with observer
         """
-        for dirname in SASS_LOAD_PATHS + applicable_sass_directories():
+        for dirname in COMMON_SASS_LOOKUP_DIRECTORIES + sass_source_directories():
```

Comment:
I think the watcher should watch all lookup directories. I would change this line to:

    for dirname in sass_lookup_directores() + sass_source_directories():